### ISSUE TITLE
Problems with the Mutated Abomination at Icecrown Citadel

### ISSUE THREAD

**Issue by fernandoart** [Draztal/WoW-Elune-Tracker#2]
ID DE PROFESOR 36678 >>>>>> ID ABOMINACION 38285>>>>> spell de relentizacion 70539

In the Professor Putricide's room, when any of the players clicked the professor's table he or she becomes into a Mutated Abomination(npcid=38285). The problem beging when in heroic mode, in the interfases the player acomulate 90 to 100% of energy with the mutated abomination to reduce the speed movement of the Gas Cloud(npcid=37562) and the Volatile Ooze(npcid=37697) with the mutated abomination's spell Regurgitated Ooze(spellid=70539) that reduce the energy of the mutated abomination en 45 points.

In other words if tha abomination has 90 to 100 energy points, and use the power only waste 45 energy points. The thing is that when the player click the power to do this action, is like as he was clicking twise and that its imposible because on click use 2 times that power so one gas cloud or volatile ooze will move faster and ther raid will wipe . thanks! :D o hope you understand :D
